src/azure-cli/azure/cli/command_modules/sql/commands.py

```diff
@@ -270,6 +271,23 @@ def load_command_table(self, _):
         g.generic_update_command('update', custom_func_name='server_audit_policy_update', supports_no_wait=True)
         g.wait_command('wait')
 
+    server_dev_ops_audit_settings_operations = CliCommandType(
+        operations_tmpl='azure.mgmt.sql.operations#ServerDevOpsAuditSettingsOperations.{}',
+        client_factory=get_sql_server_dev_ops_audit_settings_operations)
+
+    with self.command_group('sql server ms-support audit-policy',
```

Comment:
would you mind adding `is_preview=True` for your new command group if it is not ready to GA?

Reply:
This feature is ready for GA which will be released in 4-5 weeks from now.
Do you think I still need to set 'is_preview=True'?

Reply:
If there is no changes for your server side and CLI interface, you could not add it.

Reply:
Thank you